doc/requirements.txt

```diff
@@ -30,6 +30,6 @@ lxml_html_clean
 
 # Pre-install PL development wheels
 --extra-index-url https://test.pypi.org/simple/
-pennylane-lightning-kokkos==0.40.0-dev14
-pennylane-lightning==0.40.0-dev14
+pennylane-lightning-kokkos==0.40.0-dev22
+pennylane-lightning==0.40.0-dev22
```

Comment:
update to the correct testpypi version once https://github.com/PennyLaneAI/pennylane-lightning/pull/1003 is merged in lightning

Reply:
Done